lms/djangoapps/verify_student/services.py

```diff
@@ -244,6 +253,9 @@ def get_verification_details_by_id(cls, attempt_id):
         If the verification object cannot be found, returns None
         """
         verification = None
+
+        # TODO: Not updated for new model since we can't query multiple tables
```

Comment:
should we consider this function out of scope for this ticket? We can't refactor or remove it until we change how the support tools work.

Reply:
That makes sense to me, especially given that this is dependent on changes elsewhere?

Reply:
I think so, yes. However, it's possible Axim may not approve this without us either fixing it or describing our plan to fix it in detail, so let's see how our conversation goes tomorrow and go from there. I think there's no harm putting it up for review - we could get some additional feedback about how we could solve this.

Reply:
[Update] We've decided to move ahead with this PR. I've left a comment on this function detailing that it has not been updated to account for the new model and referenced the DEPR here: https://github.com/openedx/edx-platform/issues/35452